snapcraft.yaml

```diff
@@ -1241,7 +1241,7 @@ parts:
     build-environment: *buildenv
     build-packages:
       - dbus
-      - libmozjs-91-dev
+      - libmozjs-102-dev
```

Comment:
This package isn't available in core22 repositories.

Reply:
It's available in Jammy repos.
https://packages.ubuntu.com/jammy/libmozjs-102-dev

Reply:
Mmm... but I added it the snapcraft.yaml file, and snapcraft complains...

Reply:
Odd... now it does work.

Reply:
Should I add a patch to the `meson.build` removing the `have_gtk4` variable and update `gjs`?

Reply:
libmozjs-102-dev is in jammy-updates and jammy-security, maybe snapcraft isn't pulling from there?

Reply:
I don't know what happened... I began a new build with the three changes (the GJS version and the libmozjs-102-dev), and snapcraft complained that libmozjs-102-dev didn't exist. I reverted only libmozjs, and it worked (but failed in GJS because it was the wrong libmozjs version). I changed the libmozjs package to 102 again and continued the build (this is, didn't use a `clean`) and it worked.

Now it works even after a "clean", but snapcraft was also updated in the meantime, so maybe it was a bug, or maybe it was something that I did wrong, I don't know.

Reply:
About the patch, maybe the best way is to check how is `master` fixing the bug instead of just removing that.

Reply:
Master removes that variable

Reply:
Ok. Prepare the patch, then.

Reply:
Hiii, @sergio-costas @kenvandine sorry, it took me some time, but the patch for the gjs `1.76.2` is here, and seems like, it's working fine.

Reply:
@soumyaDghosh Thanks! I'll test it right now, but looks great.

Reply:
Ok, it builds perfectly, so LGTM.